src/openms/source/ANALYSIS/OPENSWATH/MRMDecoy.cpp

```diff
@@ -116,8 +124,8 @@ namespace OpenMS
   }
 
   OpenMS::TargetedExperiment::Peptide MRMDecoy::shufflePeptide(
-    OpenMS::TargetedExperiment::Peptide peptide, double identity_threshold, int seed,
-    int max_attempts)
+    OpenMS::TargetedExperiment::Peptide peptide, const double identity_threshold, int seed,
+    const int max_attempts)
```

Comment:
yes, that could be the next step. Since these methods are more general than just for MRM decoys, that should be the next step. However, we have to think about whether we then still want to use `OpenMS::TargetedExperiment::Peptide` or switch to `AASequence`